Lambda-Homework/src/lambda/homework/Person.java

```diff
@@ -0,0 +1,131 @@
+package lambda.homework;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Person {
+
+    private String firstName;
+    private String lastName;
+    private LocalDate date;
+
+    public Person(String firstName, String lastName, LocalDate date) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.date = date;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public LocalDate getDate() {
+        return date;
+    }
+
+    @Override
+    public String toString() {
+        return "Person{" +
+                "firstName='" + firstName + '\'' +
+                ", lastName='" + lastName + '\'' +
+                ", date=" + date +
+                '}';
+    }
+
+    /**
+     * the method read the information from a file and put them in a list
+     * @param filePath the string that contains the path of te file to read from
+     * @return the list with the information read from the file
+     */
+
+    public static List<String> readFromFile(String filePath) {
+        List<String> personsInfo = new ArrayList<>();
+
+        BufferedReader file1 = null;
+        try {
+            file1 = new BufferedReader(new InputStreamReader(new FileInputStream(filePath)));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
exceptia aceasta se poate arunca mai departe in cazul asta, pentru a semnala ca exista aceasta posibilitate.